### PR TITLE
Forgot password,signup and skip button color changed for visibility. …

### PR DESCRIPTION
**Pull request by anuranBarman**
…Issue #491

Fixes #[491]

Changes: [changed tint color of the three button and programmatically set title color of the buttons.]

### CODE REVIEW

File: Susi.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -764,7 +764,7 @@
 					};
 					6D4D9B361E768E190067459C = {
 						CreatedOnToolsVersion = 8.2.1;
-						DevelopmentTeam = 7P4K4M8A4Q;
+						DevelopmentTeam = 29HRCWU889;
```
Comment (chashmeetsingh) [inline Susi.xcodeproj/project.pbxproj]: Changes to this file aren't required.

Comment (anuranBarman) [reply Susi.xcodeproj/project.pbxproj]: could not build it without it for testing due to profile error.

File: Susi/Controllers/ResetPasswordController/ResetPasswordVCMethods.swift [hunk 2]
```diff
@@ -9,7 +9,7 @@
 import UIKit
 import Material
 
-extension ResetPasswordViewController {
+extension ResetPasswordViewController:UITextFieldDelegate {
```
Comment (jogendra) [inline Susi/Controllers/ResetPasswordController/ResetPasswordVCMethods.swift]: Follow the Swift guidelines mentioned in the README. There should be a space after `:`

File: Susi/Controllers/ResetPasswordController/ResetPasswordVCMethods.swift [hunk 3]
```diff
@@ -23,6 +23,10 @@ extension ResetPasswordViewController {
         if let navbar = navigationController?.navigationBar {
             navbar.barTintColor = UIColor.defaultColor()
         }
+        
+        self.currentPasswordField.delegate=self
+        self.newPasswordField.delegate=self
+        self.confirmPasswordField.delegate=self
```
Comment (jogendra) [inline Susi/Controllers/ResetPasswordController/ResetPasswordVCMethods.swift]: same here. there should be space before and after `=`

**Comment by jogendra** [comment]
@anuranBarman Closing this PR for now as no activity from your side. @baquer @raghavbabbar can anyone of you take this issue?

**Comment by baquer** [comment]
@jogendra , i am taking this issue.

**Comment by jogendra** [comment]
@baquer Great! looking forward to your PR

**Pull request closed without merging**